chain/chain.go

```diff
@@ -55,6 +55,9 @@ type Genesis struct {
 	BaseFee    uint64                            `json:"baseFee"`
 	BaseFeeEM  uint64                            `json:"baseFeeEM"`
 
+	// The initial prices to be set for the Price module
+	InitialPrices [310]*big.Int `json:"initialPrices"`
```

Comment:
There is no reason to enter the prices in the genesis block. They must be part of the genesis config file only.